### ISSUE TITLE
Trending Projects / New Releases are not represented in semantic HTML

### ISSUE THREAD

**Issue by SteinRobert** [pypa/warehouse#4343]
**Describe the bug**
On the index page trending projects and new releases are not represented in semantic HTML, which should be used in the project as stated in the ui principles.
```
<div class="package-list">
    <div class="package-snippet">...</div>
    <div class="package-snippet">...</div>
    ...
</div>
```
Even though the CSS class name contains `list` the representation of itself are simply `<div>` tags.

**Expected behavior**
This should be an ordered list:
```
<ol class="package-list">
    <li class="package-snippet">...</li>
    <li class="package-snippet">...</li>
    ...
</ol>
```
It would be even better if the list would be labeled by the preceding headline:
```

<div class="heading-wsubtitle" id="pypi-trending-packages">
    <h2 class="heading-wsubtitle__heading">Trending projects</h2>
    <p class="heading-wsubtitle__subtitle">Trending projects as downloaded by the community</p>
</div>
<ol class="package-list" aria-labeledby="pypi-trending-packages">
    <li class="package-snippet">...</li>
    <li class="package-snippet">...</li>
    ...
</ol>
```
**To Reproduce**
Just take a look at the current index page.

**My Platform**
Does not matter in this case.

**Additional context**
For modern screen readers like NVDA or JAWS the correct semantics of an HTML document are essential. 
In this case a screenreader _could_ announce "Trending Packages - List of 5 Items - First Item <Package Name>" *pressing Tab* "Second Item <Package Name>".

*Hope I got this right. Let me know if you agree or disagree with my suggestions - I'd love to discuss the issue and provide a PR if needed.*

**Comment by di** [comment]
Seems like a reasonable change. Would you like to submit a PR?

**Comment by SteinRobert** [comment]
Sure, just give me a few days.

**Comment by ejweavers** [comment]
@nlhkabu can i pick this up in the europython2018 sprint?

**Comment by nlhkabu** [comment]
Yes, that will be great @ejweavers! I will assign it to myself (as I can't assign to you), but consider it reserved :)
